### DISCUSSION TITLE
可以读取IDEA的窗口标题代替监听软件名称吗？

### DISCUSSION THREAD

**Discussion by kakapoZHCN** [shion-app/shion#6] [Ideas]
我有一个想法，监听软件时有可能打开多个相同程序，我想要区分使用的是哪个。比如IDEA打开多个项目，可以读取窗口标题做个区分吗。

**Comment by hanaTsuk1** [comment]
使用窗口标题做标识区分，实在不能算作一个通用的方案
很多软件的窗口标题会做细致的变化，导致一个软件下可以分出很多个子类，十分杂乱
例如浏览器
